docs/change-log/index.md

```diff
@@ -546,29 +577,97 @@ with a drill-down to view the steps and their details. [Tech Preview]
+- Five runtime legacy REST APIs, such as: `list_runtime_resources_legacy`, `delete_runtime_object_legacy` etc.
+- httpdb runtime-related APIs using the deprecated runtimes REST APIs, for example: `delete_runtime_object`
+
+
+## Future Deprecations
+
+
+| ID   | Description                                                    |
+| --- | ----------------------------------------------------------------- |
+| ML-3605 | Model Monitoring:  Most of the charts and KPIs in Grafana are now based on the data store target instead of the MLRun API. It is recommended to update the model monitoring dashboards since the old dashboards won't be supported in v1.5.0. |
+| ML-4171 | In v1.4.1 Redi targets will have one key per record. This will not be backwards-compatible. |
+
+
+
+### Deprecated APIs, will be removed in v1.5.0
+These APIs will be removed from the v1.5.0 code. A FutureWarning appears if you try to use them in v1.3.0.
+| Deprecated / to be removed                       | Use instead                                   |
+| ------------------------------------------------ | --------------------------------------------- |
+| project-related parameters of `set_environment`. (Global-related parameters will not be deprecated.) | The same parameters in project-related APIs, such as `get_or_create_project` |
+| `KubeResource.gpus`                              | `with_limits`                 |
+| Dask `gpus`                                      | `with_scheduler_limits` / `with_worker_limits`   |
+| `ExecutorTypes`                                  | `ParallelRunnerModes`         |
+| Spark runtime `gpus`                              | `with_driver_limits` / `with_executor_limits` |
+| `mount_v3io_legacy` (mount_v3io no longer calls it) | `mount_v3io`                       |
+| `mount_v3io_extended`                            | `mount_v3io`                   |
+| `LegacyArtifact` and all legacy artifact types that inherit from it (`LegacyArtifact`, `LegacyDirArtifact`, `LegacyLinkArtifact`, `LegacyPlotArtifact`, `LegacyChartArtifact`, `LegacyTableArtifact`, `LegacyModelArtifact`, `LegacyDatasetArtifact`, `LegacyPlotlyArtifact`, `LegacyBokehArtifact`) | `Artifact` or other artifact classes that inherit from it |
+| `init_functions` in pipelines                    | Add the function initialization to the pipeline code instead |
+| The entire `mlrun/mlutils` library               | `mlrun.framework`                     |
+| `run_pipeline`                                   | `project.run`                                     |
+
+### Deprecated APIs, will be removed in v1.6.0
+These APIs will be removed from the v1.6.0 code. A FutureWarning appears if you try to use them in v1.6.0.
+| Deprecated / to be removed                       | Use instead                                   |
+| ------------------------------------------------ | --------------------------------------------- |
+| clear_context() |  |
+
+
+
+### Deprecated CLI, will be removed in v1.5.0
```

Comment:
As I see it this section should be above the "will be removed in v1.6.0" one

Reply:
I grouped them by API/CLI and not release. Both work - but switched them